friends/src/components/FriendList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+
+const FriendList = (props) => {
+    return(
+        <div className="friends-list">
+        <h1>Friends</h1>
+            {props.friendList.map(friend => {
+                // Need to find a better key value
+                return(
+                    <h2 key={friend.name}>{friend.name}</h2>
```

Comment:
I think you want to get the email and age to display too possibly, at least somewhere in the program